lib/tasks/mastodon.rake

```diff
@@ -1,6 +1,16 @@
 # frozen_string_literal: true
 
 namespace :mastodon do
+  desc 'Execute daily tasks'
+  task :daily do
```

Comment:
e.g. Right now, running `rake mastodon:daily` returns no relevant output. It would be nicer if it had something like:

```
Clearing feeds...
Refreshing PuSH subscriptions...
```

If it was easy enough to do, could you perhaps put the list of tasks into an array, and iterate over it, printing the task `desc` before each one?